editorial_material/editorial_policies.tex

```diff
@@ -0,0 +1,84 @@
+
+\maketitle
+
+\begin{abstract}
+The current draft list of editorial board policies for the Living
+Journal of Computational Molecular Science (LiveCoMS). These policies
+are not yet active, as they are still in draft.
+\end{abstract}
+
+\section{Policies on Submissions to LiveCoMS}
+
+\subsection{Policy on Licensing}
+
+LiveCoMS does not ask for or request any copyright transfer.
+
+However, in order to submit to LiveCoMS, authors must provide, at
+minimum, a license for LiveCoMS to publish the article and distributed
+it free of charge. We recommend that the authors release the article
+under an open source license such as \href{Creative Commons
+  Attribution}{https://creativecommons.org/licenses/by/4.0/},
+releasing the document for anyone to copy and redistribute the
+material in any medium or format, and remix, transform, and build upon
+the material for any purpose, even commercially.  Making it available
+to all of course makes it possible for LiveCoMS to publish it, and for
+the community to edit and contribute.
+
+Other more restrictive licenses may be possible as long as LiveCoMS
+has the permission to publish and excerpt from the document; this
+might be required if someone other than the authors has some rights to
+the material (for example, if it was previously published in some
+other journal). Generally, we only allow a more restrictive license in
+these cases, but are happy to discuss. Please ask the managing editors
+if you require some other licensing regime.
+
+\subsection{Policies on Author Order}
+Authors should, to the extent possible, determine the author order among themselves. 
+
+\subsection{Policy on Abandoned Documents}
+
+All submitters agree by submission of the document to so that if
+authors die or abandon an article that the community wants to
+continue, they have already given consent for us to come up with a way
+to continue it.  In most cases, this will be addressed explicitly by
+the license that the document is submitted to LiveCoMS. Generally, it
+will be submitted under a license like \href{Creative Commons
+  Attribution}{https://creativecommons.org/licenses/by/4.0/}, in which
+case it is clear that anyone can reuse and ``resurrect'' the document.
+
+However, as a case of professional and academic courtesy, authors
+should At the same time, there are still issues of ``academic
+courtesy''. For this reason, if the document has not been updated by
+one of the authors, and they do not respond, submission to LiveCoMS
+means that the authors agree that the other others may continue. If no
+authors are contactable, then LiveCoMS may accept an article based on
+a previous LiveCoMS article as long as the previous authors are
+recognized. 
+
+\section{Policies About the Editorial Boards}
+
+\subsection{Policies on Decisions}
+Issues regarding governance structure, editorial board membership, leadership, and financial
+model must be decided by a 2/3 majority vote of the editorial
```

Comment:
"2/3 majority vote of the voting members of the editorial board"? Just to make clear we're not saying 2/3 of everyone but 2/3 of those voting....

Reply:
Done.